app/forms/provider_interface/offer_wizard.rb

```diff
@@ -121,14 +124,19 @@ def self.standard_conditions_from(offer)
 
     private_class_method :standard_conditions_from
 
-    def self.further_conditions_from(offer)
-      return [] if offer.blank?
+    def self.further_condition_attrs_from(offer)
+      return {} if offer.blank?
 
-      conditions = offer.conditions.map(&:text)
-      conditions - MakeOffer::STANDARD_CONDITIONS
+      further_conditions = offer.conditions.reject do |condition|
+        MakeOffer::STANDARD_CONDITIONS.include?(condition.text)
```

Comment:
Potential refactoring: Add `standard_condition?` method to OfferCondition, and also move the STANDARD_CONDITIONS constant to the model, as it seems to be a more appropriate place.
(Not for this PR)

Reply:
Yes I like that idea 👍